lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -392,9 +402,10 @@ static void addPerfEarlyModulePassPipeline(SILPassPipelinePlan &P) {
   // we do not spend time optimizing them.
   P.addDeadFunctionElimination();
 
-  // Strip ownership from non-transparent functions.
-  if (P.getOptions().StripOwnershipAfterSerialization)
-    P.addNonTransparentFunctionOwnershipModelEliminator();
+  // Strip ownership from non-transparent functions that are in our module.
+  if (P.getOptions().StripOwnershipAfterSerialization) {
+    P.addLocalNonTransparentOwnershipModelEliminator();
```

Comment:
According to the description, this strips a strict subset of what was stripped earlier by `addLocalOwnershipModelEliminator`. So, I'm not really sure what this pass actually does here.

...unless this actually means strip local *or* non-transparent functions.